source/MaterialXRuntime/RtFileIo.cpp

```diff
@@ -491,6 +493,25 @@ namespace
         }
     }
 
+    void writeSourceUris(PrvStage* stage, DocumentPtr doc)
```

Comment:
Stage scan to write optional xincludes. Needed if you want to load in a document with dependent definitions / nodes as our load checks against the current MaterialX document and not the Stage contents.

Reply:
I think `stage` could be a const pointer here?